pkg/application/application.go

```diff
@@ -114,6 +114,7 @@ func AddFlags(cmd *cobra.Command) *CreateOptions {
 	cmd.Flags().StringVar(&opts.DestNamespace, "dest-namespace", "", "K8s target namespace (overrides the namespace specified in the kustomization.yaml)")
 	cmd.Flags().StringVar(&opts.InstallationMode, "installation-mode", InstallationModeNormal, "One of: normal|flat. "+
 		"If flat, will commit the application manifests (after running kustomize build), otherwise will commit the kustomization.yaml")
+	cmd.Flags().StringToStringVar(&opts.Labels, "labels", nil, "Optional labels that will be set on the Application resource. (e.g. \"{{ placeholder }}=my-org\"")
```

Comment:
Not sure I understand the example here, the `(e.g. \"{{ placeholder }}=my-org\"` part. Which probably means no one will understand it. We need to think of a better way to explain or to just drop the explanation here and leave it for the docs.

Reply:
yeah, you are right. i'll add a special md file to document how to use labels in `project create` and then in `app create`.